### PR TITLE
🐛 Fix so a update on message template creates a change

### PR DESCRIPTION
**Pull request by davidsoderberg**

### CODE REVIEW

File: apps/web/cypress/tests/notifications-editor.spec.ts [hunk 1]
```diff
@@ -191,12 +191,11 @@ describe('Notifications Creator', function () {
 
   it('should validate form inputs', function () {
     cy.visit('/templates/create');
-    cy.getByTestId('submit-btn').click();
-
-    cy.getByTestId('title').should('have.class', 'mantine-TextInput-invalid');
     addChannel('inApp');
+    cy.getByTestId('description').type('this is a notification template description');
```
Comment (ainouzgali) [inline apps/web/cypress/tests/notifications-editor.spec.ts:195]: I think you should change the order of addChannel and typing into description. I think that should fix the test

**Review by ainouzgali — COMMENTED** [review]
Looks good, I love that you fixed the empty content. So much cleaner then what we did before

File: apps/web/src/pages/templates/editor/TemplateEditorPage.tsx [hunk 2]
```diff
@@ -106,7 +107,7 @@ export default function TemplateEditorPage() {
                     mr={20}
                     data-test-id="submit-btn"
                     loading={isLoading || isUpdateLoading}
-                    disabled={readonly || loadingEditTemplate || isLoading}
+                    disabled={readonly || loadingEditTemplate || isLoading || !isDirty}
```
Comment (ainouzgali) [inline apps/web/src/pages/templates/editor/TemplateEditorPage.tsx:110]: Should we add an error message to explain why it is disabled? For example, the user adds a subject in email, but no content. It is generally allowed and would not trigger our errors that we show on clicking submit. I'm not sure if its clear, wdyt?

**Review by ainouzgali — APPROVED** [review]

File: apps/web/src/pages/templates/editor/TemplateEditorPage.tsx (hunk 2 above)
Comment (davidsoderberg) [reply apps/web/src/pages/templates/editor/TemplateEditorPage.tsx:110]: Like not just because we add the not dirty check but in general maybe...

Comment (ainouzgali) [reply apps/web/src/pages/templates/editor/TemplateEditorPage.tsx:110]: All good :) Thanks for checking

**Pull request merged**